### DISCUSSION TITLE
zyklisches Senden - was ist damit genau gemeint?

### DISCUSSION THREAD

**Discussion by Kenaschon** [dewenni/ESP_Buderus_KM271#134] [General]
In den mqtt-Einstellungen gibt es den Eintrag zyklisches Senden". Werden alle Daten dann im eingestelltem Intervall gesendet?

Grüße
Manfred

**Comment by dewenni** [comment]
Ja genau. Kann hilfreich sein wenn man die Daten z.b in eine Datenbank schreibt und über Grafana anzeigt. Es gibt Werte die ändern sich nur alle paar Tage mal und dann fehlt der Wert dort in der Anzeige wenn man z.b nur 1 Tag anzeigt.

**Comment by Kenaschon** [comment]
Danke.

**Reply by dewenni** [reply]
@Kenaschon btw. ich habe gesehen, dass das hier auch interessant seien könnte für dich ;-)
https://github.com/dewenni/ESP32-Jarolift-Controller

**Comment by Kenaschon** [comment]
Jepp, ist es. Bin dann mal da.
